project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.8.3
+sbt.version = 1.9.4
```

Comment:
Will it cause deprecation warnings (errors) due to the usage of integration test config?

Reply:
Yes, it does trigger warnings